designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputNegativeInteger.kt

```diff
@@ -35,15 +39,20 @@ fun InputNegativeInteger(
     imeAction: ImeAction = ImeAction.Next,
     modifier: Modifier = Modifier,
 ) {
+    var inputValue by rememberSaveable { mutableStateOf(inputText ?: "") }
```

Comment:
does this remember be updated every time the inputText changes?

Reply:
it is updated in order to maintain the non negative value inside the input, do you think there is a better way of doing it?